src/main/java/com/hedvig/productPricing/campaign/service/exception/NoHedvigForeverCampaignException.kt

```diff
@@ -0,0 +1,5 @@
+package com.hedvig.productPricing.campaign.service.exception
+
+import com.hedvig.contract.shared.models.objects.Market
+
+class NoHedvigForeverCampaignException(market: Market) : RuntimeException("No hedvig forever campaign for $market")
```

Comment:
Super subjective but I personally dislike having marketing words in class names etc. the concept and the market name should be spearated IMO. 
```suggestion
class NoForeverCampaignException(market: Market) : RuntimeException("No hedvig forever campaign for $market")
```

Reply:
In my opinion, `ForeverCampaign` becomes something completely different. So either go with `HedvigForever` or rename the entire feature to `EnableReferralsCampaign`. I like going with `HedvigForever`